go-controller/pkg/util/pod_annotation_unit_test.go

```diff
@@ -211,6 +218,10 @@ func TestUnmarshalPodAnnotation(t *testing.T) {
 			desc:        "verify successful unmarshal of pod annotation when *only* the MAC address is present",
 			inpAnnotMap: map[string]string{"k8s.ovn.org/pod-networks": `{"default":{"mac_address":"0a:58:fd:98:00:01"}}`},
 		},
+		{
+			desc:        "verify successful unmarshal of pod annotation with ipv6 lla gateway ip",
+			inpAnnotMap: map[string]string{"k8s.ovn.org/pod-networks": `{"default":{"mac_address":"0a:58:fd:98:00:01","ipv6_lla_gateway_ip":"fe80::"}}`},
```

Comment:
could we maybe align the MAC address w/ the LLA address ? Keep in mind we only support this feature for MAC generated LLAs.

I think it'll make the test "look" better, and allow for future reference.